i18n/es.toml

```diff
@@ -72,7 +72,7 @@ other = "Ver en inglés"
 other = "Ver en español"
 
 [old_version]
-other = "Nota: The version en inglés ha sido actualizada desde está traducción"
```

Comment:
My bad here. My original comment had this mistake in it and of course since I didn't catch it it was merged. I'm going to fix it here. @tdelmas